serverless/about-serverless.adoc

```diff
@@ -0,0 +1,78 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="about-serverless"]
+= About {ServerlessProductName}
+:context: about-serverless
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+{ServerlessProductName} enables developers to deploy and manage cloud native applications that can run anywhere, without provisioning or managing servers.
+These routine tasks are abstracted away by the platform, allowing developers to push code to production much faster than in traditional models.
+
+{ServerlessProductName} is based on the open source link:https://knative.dev/[Knative] project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
```

Comment:
The way this is worded, I'm not sure whether the second part is saying that Knative project enables an enterprise-grade platform, or if it's supposed to mean that OpenShift Serverless does.